deployment/network-operator/README.md

```diff
@@ -100,6 +100,22 @@ $ helm install --devel --set nfd.enabled=false -n network-operator --create-name
 custom resources. The user is required to create it later with configuration matching the cluster or use
 chart parameters to deploy it together with the operator.
 
+## Helm Tests
+
+Network Operator has helm tests to verify deployment, to run tests it is needed to deploy the instance resource of NicClusterPolicy with `deployCR`, additionally needs to deploy `devicePlugin` and `secondaryNetwork` on executing `install/upgrade`.
+Supported Tests:
+- Device Plugin Resource: This test creates a pod that requests the first resource in `devicePlugin.resources`
+- RDMA Traffic: This test creates a pod that test loopback RDMA traffic with `rping`
+
+Run the helm test with following command after deploying network operator with helm
+```
+$ helm test -n network-operator network-operator --timeout=5m
+```
+Notes:
+- Test will keeping running endlessly if pod creating failed so it is recommended to use `--timeout` which fails test after exceeding given timeout
+- Default PF to run test is `ens2f0` to override it add `--set test.pf=<pf_name>` to `helm install/upgrade`
+- Tests should be executed after `nicclusterpolicy` custom resource state is `Ready`
```

Comment:
add a note also on failure , that the test resources will not be cleaned, and how to collect logs for debug

Reply:
updated

Reply:
nit: `nicclusterpolicy` -> `NicClusterPolicy`